exchange/exchangetest/request-multi-bidders-debug-info.json

```diff
@@ -95,7 +96,8 @@
             "uri": "audienceNetworkTest.com",
             "requestbody": "audienceNetworkTestRequestBody",
             "responsebody": "audienceNetworkTestResponseBody",
-            "status": 200
+            "status": 200,
+            "requestheaders": null
```

Comment:
Added new parameter to satisfy the unit test

Reply:
Could you please group `requestheaders` closer to `requestbody` in the json files? I think it's easier to read. No need to add it to the end just because it's new.

Reply:
grouped `requestheaders` closer to `requestbody`